### PR TITLE
doc: update the following VM topics

### PR DESCRIPTION
**Pull request by Vicente-Cheng**
- Hot-plug Volumes
    - Resource Overcommit
    - Clone VM

### CODE REVIEW

File: docs/vm/resource-overcommit.md [hunk 1]
```diff
@@ -33,12 +33,25 @@ Classic memory overcommitment or memory ballooning is not yet supported by this
...
 ## Configure overcommit for a single virtual machine
 
-If you need to configure individual virtual machines without involving global configuration, consider adjusting the ` spec.template.spec.domain.resources.<memory|cpu>` value on the target VirtualMachine resource individually. Note that by modifying these values, you are taking over control of virtual machine resource management from Harvester.
+If you need to configure individual virtual machines without involving global configuration, consider adjusting the ` spec.template.spec.domain.resources.limit.<memory|cpu>` value on the target VirtualMachine resource individually.
+
+![vm overcommit config](/img/v1.2/vm/vm-overcommit-config.png)
+
+:::note
+
+After modifying these values, you are taking over control of virtual machine resource management from Harvester.
```
Comment (vickyhella) [inline docs/vm/resource-overcommit.md:52]:
```suggestion
After modifying these values, you can manage the virtual machine resources from Harvester.
```

File: docs/vm/resource-overcommit.md [hunk 2]
```diff
@@ -33,12 +33,25 @@ Classic memory overcommitment or memory ballooning is not yet supported by this
...
+
 1. Find the `overcommit-config` setting.
 1. Configure the desired CPU, Memory, and Storage ratio.
 
+![overcommit panel](/img/v1.2/vm/overcommit-panel.png)
+
 ## Configure overcommit for a single virtual machine
 
-If you need to configure individual virtual machines without involving global configuration, consider adjusting the ` spec.template.spec.domain.resources.<memory|cpu>` value on the target VirtualMachine resource individually. Note that by modifying these values, you are taking over control of virtual machine resource management from Harvester.
+If you need to configure individual virtual machines without involving global configuration, consider adjusting the ` spec.template.spec.domain.resources.limit.<memory|cpu>` value on the target VirtualMachine resource individually.
```
Comment (vickyhella) [inline docs/vm/resource-overcommit.md:46]:
```suggestion
If you need to configure individual virtual machines without involving global configuration, consider adjusting the ` spec.template.spec.domain.resources.limit.<memory|cpu>` value on the target virtual machine resource individually.
```

**Review by vickyhella — COMMENTED** [review]
Only some nits

File: docs/vm/clone-vm.md [hunk 3]
```diff
@@ -27,6 +27,8 @@ VM can be cloned with/without data. This function doesn't need to take a VM snap
 
 ## Clone VM without volume data
 
+This operation more like to create a new VM with the same configuration as the source VM.
```
[the remark this answers is not in the document]
Comment (Vicente-Cheng) [reply docs/vm/clone-vm.md]: updated! thanks!!

File: docs/vm/resource-overcommit.md (hunk 1 above)
Comment (Vicente-Cheng) [reply docs/vm/resource-overcommit.md]: updated! thanks!

**Review by vickyhella — APPROVED** [review]

**Review by bk201 — APPROVED** [review]
LGTM, thanks!

File: docs/vm/resource-overcommit.md [hunk 4]
```diff
@@ -33,12 +33,25 @@ Classic memory overcommitment or memory ballooning is not yet supported by this
...
-If you need to configure individual virtual machines without involving global configuration, consider adjusting the ` spec.template.spec.domain.resources.<memory|cpu>` value on the target VirtualMachine resource individually. Note that by modifying these values, you are taking over control of virtual machine resource management from Harvester.
+If you need to configure individual virtual machines without involving global configuration, consider adjusting the ` spec.template.spec.domain.resources.limits.<memory|cpu>` value on the target virtual machine resource individually.
+
+![vm overcommit config](/img/v1.2/vm/vm-overcommit-config.png)
+
+:::note
+
+After modifying these values, you can manage the virtual machine resources from Harvester.
+
+:::
```
Comment (guangbochen) [inline docs/vm/resource-overcommit.md:54]: consider removing this instead.

File: docs/vm/resource-overcommit.md [hunk 5]
```diff
@@ -33,12 +33,25 @@ Classic memory overcommitment or memory ballooning is not yet supported by this
...
+
 1. Find the `overcommit-config` setting.
 1. Configure the desired CPU, Memory, and Storage ratio.
 
+  ![overcommit panel](/img/v1.2/vm/overcommit-panel.png)
+
 ## Configure overcommit for a single virtual machine
 
-If you need to configure individual virtual machines without involving global configuration, consider adjusting the ` spec.template.spec.domain.resources.<memory|cpu>` value on the target VirtualMachine resource individually. Note that by modifying these values, you are taking over control of virtual machine resource management from Harvester.
+If you need to configure individual virtual machines without involving global configuration, consider adjusting the ` spec.template.spec.domain.resources.limits.<memory|cpu>` value on the target virtual machine resource individually.
```
Comment (guangbochen) [inline docs/vm/resource-overcommit.md:46]:
```suggestion
In situations where you require specific configurations for individual virtual machines without affecting the global settings, you can easily achieve this by modifying the `spec.template.spec.domain.resources.limits.<memory|cpu>` value on the corresponding virtual machine spec directly.
```

File: docs/vm/clone-vm.md [hunk 6]
```diff
@@ -27,6 +27,8 @@ VM can be cloned with/without data. This function doesn't need to take a VM snap
 
 ## Clone VM without volume data
 
+This operation is more like creating a new VM with the same configuration as the source VM.
```
Comment (LucasSaintarbor) [inline docs/vm/clone-vm.md:30]:
```suggestion
Cloning a VM without volume data creates a new VM with the same configuration as the source VM.
```

**Review by LucasSaintarbor — COMMENTED** [review]
Small NIT

**Review by guangbochen — APPROVED** [review]
LGTM, thanks.

**Review by LucasSaintarbor — APPROVED** [review]
LGTM, thanks

**Pull request merged**